src/firebase-app.js

```diff
@@ -0,0 +1,49 @@
+import firebase from '@firebase/app';
+import 'firebase/firestore';
+
+// Imports a module without throwing and error
+const tryRequire = (path) => {
+    try {
+        return require(`${path}`);
+    } catch (err) {
+        return null;
+    }
+};
+
+// A config file that contains your firebase project credentials (not included in the repo)
+const CONFIG = tryRequire('./firebase-config.json') || {
```

Comment:
These are your configs, do they need to be committed?

Reply:
These are just a config for a test account so that the build doesn't fail, the contributors have the option to make a firebase-config.json and place their own credentials. I also added it to the .gitignore file.

Reply:
Cool, thanks!